cypress/e2e/smoke/load_safe.cy.js

```diff
@@ -60,4 +60,33 @@ describe('[SMOKE] Load Safe tests', () => {
     safe.veriySidebarSafeNameIsVisible(testSafeName)
     safe.verifyOwnerNamePresentInSettings(testOwnerName)
   })
+
+  it('[SMOKE] Verify safe name has a default name', () => {
+    createwallet.verifyDefaultWalletName(createwallet.defaltSepoliaPlaceholder)
```

Comment:
typo: 
change "defaltSepoliaPlaceholder" for "defaultSepoliaPlaceholder"

Reply:
Done